server/webdriver_tests/timeline_test.py

```diff
@@ -0,0 +1,47 @@
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import unittest
+import urllib
+from base_test import WebdriverBaseTest
+
+TIMELINE_URL = '/tools/timeline'
+
+class TestCharts(WebdriverBaseTest):
```

Comment:
Same - rule of thumb is each new file / class should have a comment.

Reply:
Done, thanks for your comments!